src/Validus/Validus.fs

```diff
@@ -518,13 +527,10 @@ module Validators =
 /// Custom operators for ValidationResult
 module Operators =
     /// Alias for ValidationResult.apply
-    let inline (<*>) f x = ValidationResult.apply f x
-
-    /// Alias for Result.map
-    let inline (<!>) f x = Result.map f x
+    let inline (<*>) applier result = ValidationResult.apply applier result
 
-    /// Alias for ValidationResult.bind
-    let inline (>>=) x f = Result.bind f x
+    /// Alias for Validator.chain
+    let inline (<|>) v1 v2 = Validator.chain v1 v2
```

Comment:
Maybe `>=>` is the right one?
https://fsharpforfunandprofit.com/posts/recipe-part2/

Reply:
I think as far as symbols go, the kleisli operator is likely more appropriate.